python/lsst/meas/astrom/astrometry.py

```diff
@@ -55,6 +57,11 @@ class AstrometryConfig(RefMatchConfig):
         default=0.001,
         min=0,
     )
+    doWriteOutput = pexConfig.Field(
```

Comment:
I agree that this should be changed. This `Task` does not do any writing, so the name is misleading.